stories/documentation/structure/box/box-arrow.stories.ts

```diff
@@ -104,6 +104,13 @@ function getTemplate(args: ArrowBasicStory): string {
 const Template: Story<ArrowBasicStory> = (args: ArrowBasicStory) => ({
 	props: args,
 	template: getTemplate(args),
+	styles: [
+		`
+		:host {
+			display: block;
+		}`,
+		args.grey === true ? ':host { background-color: white; margin: -15px -15px; padding: 15px 15px; }' : '',
```

Comment:
Why not just `1rem`?

Reply:
These values are related to SB spacings.